packages/scripts/src/generate-readme/generator.ts

```diff
@@ -163,7 +175,7 @@ export class ReadmeGenerator {
       ? buildTable(tableText, paramHeaders)
       : 'There are no input parameters for this adapter.'
 
-    this.readmeText += `## Input Parameters\n\n${inputParamTable}\n\n---\n\n`
+    this.readmeText += `## Input Parameters\n\nEvery EA supports base input parameters from [this list](https://github.com/smartcontractkit/external-adapters-js/tree/develop/packages/core/bootstrap#base-input-parameters)\n\n${inputParamTable}\n\n`
```

Comment:
I would make this a relative link so it works in every branch independently:
```suggestion
    this.readmeText += `## Input Parameters\n\nEvery EA supports base input parameters from [this list](../../../core/bootstrap#base-input-parameters)\n\n${inputParamTable}\n\n`
```